src/pkg/services/m365/api/users.go

```diff
@@ -58,6 +58,7 @@ type MailboxInfo struct {
 	Language                   Language
 	WorkingHours               WorkingHours
 	ErrGetMailBoxSetting       []error
+	DeltaAvailable             bool
```

Comment:
`DeltaAvailble` is an inference on the quota issue.  I think this will be easier to support if we expose the case as its own boolean (ie: `DeltaAvailble` -> `QuotaExceeded`) and provide a func to evaluate the mailbox state, eg:
```go
func (ui UserInfo) CanMakeDeltaQueries() bool { return QuotaExceeded }
```
If a second condition comes along which also nullifies delta query usage, we would get both a consolidated check (the func) and clear identification of which cases caused the evaluation (the flags).